tests/integration/test_integration.py

```diff
@@ -1748,6 +1748,107 @@ def test_copy(client: TestClient, admin_access_token: str, internal_study_id: st
+    )
+    client.post(
+        f"/v1/studies/{parent_id}/areas",
+        json={"name": "area2", "type": "AREA"},
+    )
+
+    # Create a Variant from the Raw Study
+    var = client.post(f"/v1/studies/{parent_id}/variants", params={"name": "variant"})
+    assert var.status_code == 200
+    variant_id = var.json()
+    variant_study = client.get(f"/v1/studies/{variant_id}")
+    assert variant_study.status_code == 200
+
+    # Copy Variant as a reference study
+    client.post(f"/v1/studies/{variant_id}/copy?dest=copied&use_task=False")
+
+    all_studies = client.get("/v1/studies")
+    assert variant_study.status_code == 200
+    assert len(all_studies.json()) == 4
+
+    copied_study = client.get("/v1/studies?name=copied")
+    assert copied_study.status_code == 200
+    copied_id = next(iter(copied_study.json()))
+
+    # Check that the copied study contains all the datas
+    copied_areas = client.get(f"/v1/studies/{copied_id}/areas")
+    assert copied_areas.json() == client.get(f"/v1/studies/{parent_id}/areas").json()
+
+
+def test_copy_as_variant_with_outputs(client: TestClient, admin_access_token: str, tmp_path: Path) -> None:
+    client.headers = {"Authorization": f"Bearer {admin_access_token}"}
+
+    # Create a raw study and a variant
+    raw = client.post("/v1/studies?name=raw")
+    variant = client.post(f"/v1/studies/{raw.json()}/variants", params={"name": "variant"})
+
+    # Create a fake output file
+    output_file = tmp_path / "internal_workspace" / variant.json() / "output" / "output.txt"
+    output_file.parent.mkdir(parents=True)
+    output_file.write_text("Output data")
+
+    # Copy of the variant as a reference study
+    copy = client.post(
+        f"/v1/studies/{variant.json()}/copy",
+        params={"dest": "copied", "with_outputs": True, "use_task": True},  # type: ignore
+    )
+    client.get(f"/v1/tasks/{copy.json()}?wait_for_completion=True")
+
+    copied_study = client.get("/v1/studies?name=copied")
+    copied_id = next(iter(copied_study.json()))
+
+    # The new study must contain an output fodler with the same data as the source variant study
+    new_output_file = tmp_path / "internal_workspace" / copied_id / "output" / "output.txt"
+    assert output_file.read_text() == new_output_file.read_text()
+
+
+def test_copy_variant_with_specific_path(client: TestClient, admin_access_token: str, tmp_path: Path) -> None:
+    client.headers = {"Authorization": f"Bearer {admin_access_token}"}
+
+    raw = client.post("/v1/studies?name=raw")
+    assert raw.status_code == 201
+    parent_id = raw.json()
+    client.post(
+        f"/v1/studies/{parent_id}/areas",
+        json={"name": "area1", "type": "AREA"},
+    )
+    client.post(
+        f"/v1/studies/{parent_id}/areas",
+        json={"name": "area2", "type": "AREA"},
+    )
+    variant = client.post(f"/v1/studies/{raw.json()}/variants", params={"name": "variant"})
+
+    dest_path = tmp_path / "internal_workspace" / "specific_path"
```

Comment:
Actually the study should not go into that path, we keep the internal studies flat, only the `folder` property should change